CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/DefaultInteractionEventHandler.java

```diff
@@ -1006,13 +1006,21 @@ private void performDrag(GraphWriteMethods wg, final Camera camera, final Point
 
         final Vector3f delta = visualInteraction.convertTranslationToDrag(camera, position, from, to);
 
+        final int xAttribute = VisualConcept.VertexAttribute.X.get(wg);
+        final int yAttribute = VisualConcept.VertexAttribute.Y.get(wg);
+        final int zAttribute = VisualConcept.VertexAttribute.Z.get(wg);
+        final int x2Attribute = VisualConcept.VertexAttribute.X2.get(wg);
+        final int y2Attribute = VisualConcept.VertexAttribute.Y2.get(wg);
+        final int z2Attribute = VisualConcept.VertexAttribute.Z2.get(wg);
+        final int cameraAttribute = VisualConcept.GraphAttribute.CAMERA.get(wg);
+        
         draggedNodeIds.forEach(vertexId -> {
-            final Vector3f currentPos = VisualGraphUtilities.getMixedVertexCoordinates(wg, vertexId);
+            final Vector3f currentPos = VisualGraphUtilities.getMixedVertexCoordinates(wg, vertexId, xAttribute, x2Attribute, yAttribute, y2Attribute, zAttribute, z2Attribute, cameraAttribute);
             currentPos.add(delta);
-            VisualGraphUtilities.setVertexCoordinates(wg, currentPos, vertexId);
+            VisualGraphUtilities.setVertexCoordinates(wg, currentPos, vertexId, xAttribute, yAttribute, zAttribute);
         });
 
-        scheduleXYZChangeOperation(wg.getVertexCount());
+        scheduleXYZChangeOperation(Ints.toArray(draggedNodeIds));
```

Comment:
Is there a reason for using the `Ints` library rather than using `draggedNodeIds.toArray(new int[])`?

Reply:
No reason, it was just the first method I found on google

Reply:
@Nova-2119 I wouldn't mind using @antares1470's suggestion. One less library use has to be good right?

Reply:
I cant seem to get it to work :/

Reply:
Alright, lets leave it then. I haven't tried myself  - if we find a way we can update it afterwards.

Reply:
I cant seem to get it to work :/

Sorry ^thought I had sent this message days ago, apparently I never clicked comment..